Content.Server/StationRecords/Systems/StationRecordsSystem.cs

```diff
@@ -49,6 +52,28 @@ private void OnPlayerSpawn(PlayerSpawnCompleteEvent args)
             return;
 
         CreateGeneralRecord(args.Station, args.Mob, args.Profile, args.JobId, stationRecords);
+
+        /*var query = EntityQueryEnumerator<SectorStationRecordComponent>();
+
+        while (query.MoveNext(out var stationGridUid, out var comp))
+        {
+            if (TryComp<StationMemberComponent>(stationGridUid, out var stationMemberComponent))
+            {
+                var stationEntityUid = stationMemberComponent.Station;
+
+                CreateGeneralRecord(stationEntityUid, args.Mob, args.Profile, args.JobId, stationRecords);
+
+                /*
+                if (TryComp<StationRecordKeyStorageComponent>(targetId, out var keyStorage)
+                && stationEntityUid != null
+                && keyStorage.Key != null)
+                {
+                    if (!TryGetRecord<GeneralStationRecord>(Key.Value, out var record))
+                        continue;
+            }
+            }
+
+        }*/
```

Comment:
With it I believe initially reason for leaving it was to make it easier to in future to upstream merges.
As the original code wiukd ecist as commwnted and in case new changes broke whole aystwm then changes could be disabled if needed.

Though there is chance that the idea behind the commented out block has changed